enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -64,23 +74,34 @@ export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
  * Default theme options, with overrides specific to the
  * Embedding SDK environment to provide nicer defaults.
  */
-export const DEFAULT_EMBEDDED_COMPONENT_THEME: MetabaseComponentTheme = merge(
-  DEFAULT_METABASE_COMPONENT_THEME,
-  {
-    table: {
-      cell: {
-        fontSize: FONT_SIZES.tableCell.em,
-        backgroundColor: "bg-white",
-      },
+export const DEFAULT_EMBEDDED_COMPONENT_THEME: MetabaseComponentTheme = merge<
+  MetabaseComponentTheme,
+  Partial<MetabaseComponentTheme>
+>(DEFAULT_METABASE_COMPONENT_THEME, {
+  table: {
+    cell: {
+      textColor: "text-primary",
```

Comment:
I don't think `text-primary` as a default works here, as I don't think the color mappings apply here. You might need to use `text-dark` instead.

Reply:
Ah we don't even need this